.github/workflows/draw-zmk.yml

```
@@ -132,3 +155,12 @@ jobs:
           commit_options: "${{ (inputs.amend_commit == true && '--amend --no-edit') || '' }}"
           push_options: "${{ (inputs.amend_commit == true && '--force-with-lease') || '' }}"
           skip_fetch: ${{ inputs.amend_commit == true }}
+
+      - name: Artifact upload
+        id: artifact-upload-step
+        uses: actions/upload-artifact@v4
```

```suggestion
        if: ( inputs.destination == 'artifact' || inputs.destination == 'both' )
        uses: actions/upload-artifact@v4
```
Is this not necessary?

---

Absolutely! I totally missed that. Very sorry!!